src/components/ComputeModal.js

```diff
@@ -1294,6 +1336,73 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     ])
   }
 
+  const renderSparkConsole = () => {
+    const { namespace, name } = getWorkspaceObject()
+
+    return div({ style: styles.drawerContent }, [
+      h(TitleBar, {
+        id: titleId,
+        title: 'Spark Console',
+        style: { marginBottom: '0.5rem' },
+        hideCloseButton: isAnalysisMode,
+        onDismiss,
+        onPrevious: () => setViewMode(undefined)
+      }),
+      p([
+        `Some of the Spark cluster components such as Apache Hadoop and Apache Spark
+         provide web interfaces. These interfaces can be used to manage and monitor cluster
+         resources and facilities, such as the YARN resource manager, the Hadoop Distributed
+         File System (HDFS), MapReduce, and Spark.`
+      ]),
+      h(SparkInterface, {
+        interfaceName: 'yarn',
```

Comment:
Magic string alert. I'd consider making an array holding all the interface name strings and then mapping them to these SparkInterface components. 

With that in mind, I'm also not sure why the caller (`ComputeModal` in this case) is responsible for passing the `synopsis`. Seems Like there should be an object of (k = interfaceName, v = synopsis) pairs in the SparkInterface file itself.